File: patterns/decoupled-notification-service/rules-js/decoupled-notification-service-sender.js
rules.JSRule({
  name: "Info Notification",
  triggers: [triggers.ItemStateChangeTrigger("SomeCondition", undefined, "ON")],
  execute: (event) => {
    items.getItem("VT_Notify_Info").postUpdate("Information: Zustand geaendert");
  }
});

File: patterns/decoupled-notification-service/rules-python/decoupled_notification_service_sender.py
from openhab import rule, Registry
from openhab.triggers import ItemStateChangeTrigger


@rule(triggers=[ItemStateChangeTrigger("SomeCondition", state="ON")])
class InfoNotification:
    def execute(self, module, input):
        Registry.getItem("VT_Notify_Info").postUpdate("Information: Zustand geaendert")
